.github/workflows/test.yml

```diff
@@ -35,12 +35,19 @@ jobs:
     strategy:
       matrix:
         python: ['3.8', '3.11']
-        django: ['3.2', '4.1']
-        wagtail: ['4.1', '4.2']
+        django: ['3.2', '4.1', '4.2']
+        wagtail: ['4.1', '4.2', '5.0', '5.1']
         include:
           - python: '3.8'
             django: '3.2'
             wagtail: '3.0'
```

Comment:
Think we can drop this since it's 3.x

Reply:
Might also be good to replace this with `tox-gh-actions`

Reply:
I'd like to resist adding tox-gh-actions as the developer may not want to do that https://github.com/cfpb/wagtail-sharing/pull/55#discussion_r919193479
I left the include in place as v3.0 is still in the test matrix.